types/agent.go

```diff
@@ -12,6 +12,7 @@ type Agent struct {
 	Token               string           `json:"token" yaml:"token"`
 	Name                string           `json:"name" yaml:"name"`
 	MachineID           string           `json:"machineID" yaml:"machineID"`
+	EnvironmentID       string           `json:"environmentID" yaml:"environmentID"`
```

Comment:
"foreign keys" are not in these models.

You can see that the project ID wasn't here either before.